cars/v1/parallelgc.ini

```diff
@@ -0,0 +1,11 @@
+[meta]
+description=Enables the G1 garbage collector
+type=mixin
+
+[config]
+base=vanilla
+
+[variables]
+use_cms_gc=false
+use_g1_gc=false
+use_parallel_gc=true
```

Comment:
Shouldn't we also add `use_parallel_gc=false` also to https://github.com/elastic/rally-teams/blob/master/cars/v1/g1gc.ini?